File: video-series-mono/src/main/java/org/jesperancinha/video/command/AddVideoSeriesCommand.java
package org.jesperancinha.video.command;

import lombok.Data;
import lombok.Value;
import org.axonframework.modelling.command.TargetAggregateIdentifier;
import org.jesperancinha.video.data.Genre;

import java.math.BigDecimal;

@Value
@Data
public class AddVideoSeriesCommand {

	@TargetAggregateIdentifier
	String id;

	String name;

	Integer volumes;

	BigDecimal cashValue;

	Genre genre;

}

File: video-series-mono/src/main/kotlin/org/jesperancinha/video/command/AddVideoSeriesCommand.kt
package org.jesperancinha.video.command

import org.axonframework.modelling.command.TargetAggregateIdentifier
import org.jesperancinha.video.data.Genre
import java.math.BigDecimal

data class AddVideoSeriesCommand(
    @TargetAggregateIdentifier
    var id: String? = null,
    var name: String? = null,
    var volumes: Int? = null,
    var cashValue: BigDecimal? = null,
    var genre: Genre? = null
)